extensions/kind/scripts/download.ts

```diff
@@ -57,21 +57,17 @@ async function download(tagVersion: string, repoPath: string, fileName: string):
   });
   let buffer;
 
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  if ((manifests.data as any).encoding && (manifests.data as any).encoding === 'base64') {
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    buffer = Buffer.from((manifests.data as any).content, 'base64');
+  if (manifests.data.encoding && manifests.data.encoding === 'base64') {
+    buffer = Buffer.from((manifests.data as ReposGetContentResponseData).content, 'base64');
```

Comment:
question: do we need the as ReposGetContentResponseData
and if yes, do we need a instanceof or any other check allowing to perform this cast ? (same for line 63)